wrappers/C_CPP/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.26)
+project(MyProject)
+
+set(CMAKE_CXX_STANDARD 11)
+set(MAKE_CXX_STANDARD_REQUIRED ON)
+
+# ---- Download REFPROP C/C++ wrappers ----------------
+set(REPO_URL "https://github.com/usnistgov/REFPROP-wrappers.git")
```

Comment:
I went back and forth on how best to do this. The purpose is to easily download the files--making a local repository was not the goal, just a side-effect. You can't download a subfolder full of files on GitHub, but most people are just going to want a single wrapper, not the whole suite of them.  Right now, the whole repo is at 65 MB, which isn't small or huge, but it might grow.

I don't see many different users contributing commits, and in that case, you'd just be cloning the repo as usual. Going this download route is really meant for the novices who just want a wrapper to use and aren't going to be looking through commit histories. Thoughts? Changing it to download the whole repo is actually simpler, but my goal was a lightweight, simpler folder of files to understand.

Reply:
I think 65 MB zipped is like 10 MB. I wouldn't sweat that at all. You could recommend the zipfile of the repo to avoid git (which is a sticking point for some people). Alternatively we could spin this out as its own repo if you like

Reply:
I changed it to clone the entire repo now.